ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bfd-2.yml

```diff
@@ -0,0 +1,24 @@
+#### router bfd ####
+
+router_bfd:
+  interval: 900
+  min_rx: 900
+  multiplier: 50
+  multihop:
+    interval: 300
+    min_rx: 300
+    multiplier: 3
+  session_snapshot_interval: 8
+  session_snapshot_interval_dangerous: true
+  sbfd:
+    local_interface:
+      name: Loopback0
+      protocols:
+        ipv4: true
+        ipv6: true
+    initiator_interval: 500
+    initiator_multiplier: 3
+    initiator_measurement_round_trip: true
+    reflector:
+      min_rx: 600
+      local_discriminator: 155.1.3.1
```

Comment:
Please remove anything but the session_snapshot_* keys from this. No need to test everything else again.